src/component/Clock.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+export class Clock extends React.Component {
+  state = {
+    date: new Date().toLocaleTimeString(),
+  };
+
+  interval = null;
+
+  componentDidMount() {
+    this.interval = setInterval(() => {
+      // eslint-disable-next-line
+      console.log(this.state.date);
+      this.setState({ date: new Date().toLocaleTimeString() });
+    }, 1000);
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.interval);
+  }
+
+  render() {
+    const { date } = this.state;
+
+    return (
+      <>
+        { date }
+      </>
```

Comment:
Don't use the fragment here. No need for it. Use more semantic tag ( 'p' or 'span')